composer.json

```diff
@@ -16,11 +16,11 @@
     },
     "require": {
         "php": ">=5.3",
-        "react/stream": "^0.4 || ^0.3",
+        "react/stream": "^1.0 || ^0.7 || ^0.6 || ^0.5 || ^0.4 || ^0.3",
```

Comment:
I wouldn't include 1.0 here, as 1.0 isn't released yet. Changes could happen that are not planned yet.